src/Components/TitleElement/TitleElement.tsx

```diff
@@ -1,17 +1,24 @@
 import styles from './TitleElement.module.css';
 import logo from '../../assets/MyStudy.png';
 import AccountLogo from './assets/account';
+import { useNavigate } from 'react-router-dom';
 
 type TitleElementProps = {
   title: string;
 };
 
 export default function TitleElement({ title }: TitleElementProps) {
+  const navigate = useNavigate();
   return (
     <div className={styles.container}>
       <img className={styles.logo} src={logo} alt="" />
       <h2 className={styles.title}>{title}</h2>
-      <AccountLogo />
+      <AccountLogo
+        onClickHandler={() => {
+          sessionStorage.clear();
+          navigate('/');
+        }}
```

Comment:
You could build a function (e.g. handleNavigate) and use the function as an argument for onClickHandler. This would make it a tiny bit more readable, but it's not neccessary.

Reply:
good idea! Implemented it👍🏻